repository/SmalltalkCI-Core.package/SCIGemStoneServerConfigSpec.class/properties.json

```diff
@@ -0,0 +1,17 @@
+{
+	"category" : "SmalltalkCI-Core",
```

Comment:
@dalehenrich I assume `SCIGemStoneServerConfigSpec` needs to be in `SmalltalkCI-Core`, because otherwise smalltalkCI is unable to load the STON? Is there any way to avoid this?